python/ray/scripts/scripts.py

```diff
@@ -2520,9 +2561,10 @@ def add_command_alias(command, name, hidden):
 cli.add_command(disable_usage_stats)
 cli.add_command(enable_usage_stats)
 add_command_alias(job_cli_group, name="job", hidden=True)
+add_command_alias(ray_logs, name="logs", hidden=False)
 cli.add_command(state_cli_list)
```

Comment:
nit: maybe we should also do `ray_list` like what you did here. `ray_xxx` is definitely a better than my `state_cli_xxx` lol